src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -237,17 +269,17 @@ public static String sanitizeForNextUrl(String url) {
         }
         return url.replace("&", "${amp}").replace("%2B", "${plus}").replace("%23", "${hash}");
     }
-    
+
     /**
-     * Recovers the URL from sanitization due to {@link #sanitizeForNextUrl}.
+     * Recovers the URL from sanitization due to {@link SanitizationHelper.sanitizeForNextUrl}.
      * In addition, any un-encoded whitespace (they may be there due to Google's
      * behind-the-screen decoding process) will be encoded again to +.
      */
-    public static String desanitizeFromNextUrl(String url) {
-        return url.replace("${amp}", "&").replace("${plus}", "%2B").replace("${hash}", "%23")
-                  .replace(" ", "+");
+    public static String desanitizeFromNextUrl(String sanitizedUrl) {
+        return sanitizedUrl.replace("${amp}", "&").replace("${plus}", "%2B").replace("${hash}", "%23")
+                .replace(" ", "+");
```

Comment:
For readability, I suggest this line start with the period in line with the one above